lib/authorizer/auth-interface.js

```diff
@@ -8,10 +8,11 @@ var _ = require('lodash'),
  * @constructs AuthInterface
  * @param {RequestAuth} auth -
  * @param {Object} protocolProfileBehavior - Protocol profile behaviors
+ * @param {Object} requester - Run options for this run
  * @return {AuthInterface}
  * @throws {Error}
  */
-createAuthInterface = function (auth, protocolProfileBehavior) {
+createAuthInterface = function (auth, protocolProfileBehavior, requester) {
```

Comment:
`requesterOptions` maybe?
```suggestion
createAuthInterface = function (auth, protocolProfileBehavior, requesterOptions) {
```